### ISSUE TITLE
Pouvoir classer un formulaire dans 2 ou n  catégories

### ISSUE THREAD

**Issue by mirda123** [pluginsGLPI/formcreator#2152]
**Describe the solution you'd like**
Pouvoir classer un formulaire dans 2 ou n  catégories ( sur la page d'accueil et dans le menu formulaire)

Ex : sharepoint est considéré par certains comme un logiciel, par d'autres comme un outil de travail collaboratif...
Ex : un logiciel de messagerie est considéré par certains comme un logiciel, par d'autres comme un outil de communication globale . ( un pb de réception de mail n'est pas forcément lié à l'outil de messagerie en lui-même) .

Une classification multi catégorie permettrait de répondre à ces divergences de points de vue et faciliterait la "trouvaille" du formulaire adéquat pour les utilisateurs. 

![2categories](https://user-images.githubusercontent.com/78594318/109132380-10569500-7754-11eb-82a1-92f89c1f5b59.jpg)

**Comment by btry** [comment]
Bonjour

En attendant qu'un sponsor se manifeste pour la fonctionnalité, vous pouvez inciter vos utilisateurs à saisir des mots clés dans la zone de recherche.

**Comment by mirda123** [comment]
Oui, j'ai effectivement complété une grosse base de mots clé pour palier à cela. 

Cependant, la faille des mots clé  est que notre catégorie 12 : AUTRE comprend un formulaire dit "générique" ( objet / descrition) car il y a toujours un cas de figure d'un problème qui ne correspond a aucun formulaire...   

et dans ce cas, comment gérer l'affichage de ce formulaire en terme de mots clé ? 
- mettre dedans tous les mots clé de tous les autres formulaires pour  qu'il apparaisse partout ? (base enorme) 
- dire ( et redire  et re-redire) aux utilisateurs de taper "générique" s'ils ne trouvent de formulaire ? (et la c'est  sur certains vont le taper direct systématiquement pour ne pas chercher)

**Comment by btry** [comment]
Avec ça :
![image](https://user-images.githubusercontent.com/14139801/109134823-c02d0200-7756-11eb-8fc8-38f9bad4a092.png)

Si cette option est active, quand la recherche ne retourne aucun résultat, le formulaire sera proposé.

**Comment by mirda123** [comment]
ahhhhh j'avais jamais vu !! j'avais jamais scrollé en dessous de en-tete en créa de formulaire.... hum... 
Je viens de l'activer, c'est formidable. Merci au développeur qui a mis ca en place.

**Comment by btry** [comment]
C'était moi :)

Du coup, je suspens le ticket, car c'est une demande qui a déjà été faite. Il ne reste plus qu'à attendre un sponsor ou une contribution.

**Comment by poupinma** [comment]
Bonjour mirda 123 
est ce que cela te dérangerai de me faire passer un export du fichier json des fformulaires materiel ou autres pour que je puisse l'installer sur mon glpi?
cela me rendrai bien service affin de pouvoir le presenter rapidement..
merci

**Comment by mirda123** [comment]
bonjour Poupinma, 
un export de quels formulaires exactement ? Dans que l'objectif ? Il faudrait me préciser les termes "materiel ou autre " et "présenter rapidement" car je n'ai pas compris.
Merci.

**Comment by poupinma** [comment]
bonjour
 je dois faire une demonstartion de glpi avec le pluging formcreator en version 2.11.2, avec la version de glpi en vers 9.5.3
et j'aurais souhaité inserer une ou plusieurs categories avec un formulaire  ..
j'avais deja cree un formulaire avec une version en 2.10.... maius je ne pas l'importer pour cause de version non compatible...

cdt

________________________________
De : mirda123 <notifications@github.com>
Envoyé : jeudi 25 février 2021 17:16
À : pluginsGLPI/formcreator <formcreator@noreply.github.com>
Cc : poupinma <manu955i@msn.com>; Comment <comment@noreply.github.com>
Objet : Re: [pluginsGLPI/formcreator] Pouvoir classer un formulaire dans 2 ou n catégories (#2152)


bonjour Poupinma,
un export de quels formulaires exactement ? Dans que l'objectif ? Il faudrait me préciser les termes "materiel ou autre " et "présenter rapidement" car je n'ai pas compris.
Merci.

**Comment by mirda123** [comment]
Je ne saisis pas bien a qui s'adresse votre démo ? vous etes developpeur sur le plug in ? 
De mon coté, malheureusement, nos formulaires contiennent des données internes à mon employeur, et pas uniquement des questios générales,  je ne peux donc pas les transmettre brut de fonderie. Pourquoi ne pas créer 2 , 3 formulaires vite fait en 2.11 pour votre démo ?

**Comment by poupinma** [comment]
en effet je commence a les refaire un par un... histoire de pouvoir presenter quelques chose...
comment faite vous pour passer d'une version form creator a une autre version sans avoir a reecrire le formaulaire?

________________________________
De : mirda123 <notifications@github.com>
Envoyé : jeudi 25 février 2021 17:32
À : pluginsGLPI/formcreator <formcreator@noreply.github.com>
Cc : poupinma <manu955i@msn.com>; Comment <comment@noreply.github.com>
Objet : Re: [pluginsGLPI/formcreator] Pouvoir classer un formulaire dans 2 ou n catégories (#2152)


Je ne saisis pas bien a qui s'adresse votre démo ? vous etes developpeur sur le plug in ?
De mon coté, malheureusement, nos formulaires contiennent des données internes à mon employeur, et pas uniquement des questios générales, je ne peux donc pas les transmettre brut de fonderie. Pourquoi ne pas créer 2 , 3 formulaires vite fait en 2.11 pour votre démo ?

**Comment by btry** [comment]
Pour importer un formulaire provenant d'une ancienne version vous devez faire un upgrade de l'ancienne version  vers 2.11.2. Avec une instance de GLPI provisoire par exemple,
